src/main/java/com/teamscale/upload/MessageUtils.java

```diff
@@ -1,4 +1,4 @@
-package com.cqse.teamscaleupload;
+package com.teamscale.upload;
 
 import java.net.InetAddress;
 import java.net.UnknownHostException;
```

Comment:
I used MessageFormat to format the string, but I did not use the date formatting capabilities since they are dependant on the system locale. So uploading from a DE system will yield a different message than uploading from EN. I believe this is not desirable (which is why I chose RFC1123 to format the date locale independently in the first place).